datafusion/functions-aggregate/src/approx_percentile_cont.rs

```diff
@@ -104,6 +105,12 @@ impl ApproxPercentileCont {
             None
         };
 
+        if args.ignore_nulls {
```

Comment:
Maybe it is worth a negative test for this (aka run a query that generates this Error)

Reply:
Removed the check / error for now